src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -69,9 +82,15 @@ public function supportsNormalization($data, $format = null)
     public function denormalize($data, $class, $format = null, array $context = array())
     {
         $dateTimeFormat = isset($context[self::FORMAT_KEY]) ? $context[self::FORMAT_KEY] : null;
+        $timezone = $this->getTimezone($context);
 
         if (null !== $dateTimeFormat) {
-            $object = \DateTime::class === $class ? \DateTime::createFromFormat($dateTimeFormat, $data) : \DateTimeImmutable::createFromFormat($dateTimeFormat, $data);
+            if (null === $timezone) {
+                // https://bugs.php.net/bug.php?id=68669
```

Comment:
As [this issue](https://github.com/php/php-src/pull/1167) is related to an unsupported version PHP, I think it's not usefull to handle this problem here.